.github/workflows/_build.yml

```diff
@@ -132,15 +132,14 @@ jobs:
 
       - name: Analyze
         run: ./scripts/ci ${{ matrix.schemes }} analysis
-
   TestPods:
     name: Testing Pods
     runs-on: macos-latest
     needs: [Setup]
     strategy:
       fail-fast: false
       matrix:
-        schemes: [uikit, swiftui]
+        schemes: [uikit]
```

Comment:
was this intentional?

Reply:
This was, as SwiftUI doesn't have a separate 'snapshot' target. 
Should we introduce that? 

UIkit: SnapshotTest / UnitTest
SwiftUI: SnapshotTest / UnitTest